homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -68,13 +68,50 @@ class SwissPublicTransportSensorEntityDescription(SensorEntityDescription):
             key=f"departure{i or ''}",
             translation_key=f"departure{i}",
             device_class=SensorDeviceClass.TIMESTAMP,
+            icon="mdi:bus-clock",
             has_legacy_attributes=i == 0,
             value_fn=lambda data_connection: data_connection["departure"],
             exists_fn=lambda data_connection: data_connection is not None,
             index=i,
         )
         for i in range(SENSOR_CONNECTIONS_COUNT)
     ],
+    SwissPublicTransportSensorEntityDescription(
+        key="duration",
+        translation_key="duration",
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        icon="mdi:timeline-clock",
+        has_legacy_attributes=False,
+        value_fn=lambda data_connection: data_connection["duration"],
+        exists_fn=lambda data_connection: data_connection is not None,
```

Comment:
What is the value if all exists_fn are the same?

Reply:
@joostlek I don't understand the question. Only for the first connection, a duration sensor is created.

Reply:
I mean, every sensor I check has the same lambda for `exists_fn` so I was wondering what the added value of `exists_fn` is

Reply:
as discussed and agreed, it does not add much value